### PR TITLE
Specify server address as a part of ROMMON location

### PR DESCRIPTION
**Pull request by annikulin**
This PR updates parameters for `ftd_install` module: it used to have a separate `tftp_server` parameter for TFTP server address, now it has changed to be a part of `rommon_file_location` parameter.

BEFORE:
```
        tftp_server: 10.89.0.11
        rommon_file_location: 'installers/ftd-boot-9.10.1.3.lfbff'
```
AFTER:
```
        rommon_file_location: 'tftp://10.89.0.11/installers/ftd-boot-9.10.1.3.lfbff'
```

### CODE REVIEW

File: test/unit/module_utils/test_device.py [hunk 1]
```diff
@@ -44,6 +49,16 @@ def test_supports_ftd_model_should_return_false_for_non_supported_models(self):
...
 
+    def test_parse_rommon_file_location(self):
+        server, path = AbstractFtdPlatform.parse_rommon_file_location('tftp://1.2.3.4/boot/rommon-boot.foo')
+        assert '1.2.3.4' == server
+        assert '/boot/rommon-boot.foo' == path
+
+    def test_parse_rommon_file_location_should_fail_for_non_tftp_protocol(self):
+        with pytest.raises(ValueError) as ex:
+            AbstractFtdPlatform.parse_rommon_file_location('http://1.2.3.4/boot/rommon-boot.foo')
+            assert 'The ROMMON image must be downloaded from TFTP server' in str(ex)
```
Comment (119Vik) [inline test/unit/module_utils/test_device.py]: shouldn't we use assert outside of with block?

File: samples/ftd_install/install_ftd_on_2110_with_local_connection.yml [hunk 2]
```diff
@@ -18,7 +18,6 @@
         console_username: console_user
         console_password: console_pass
 
-        tftp_server: 10.89.0.11
-        rommon_file_location: 'installers/fxos-k8-fp2k-lfbff.2.6.1.94i.SPA'
+        rommon_file_location: 'tftp://10.89.0.11/installers/fxos-k8-fp2k-lfbff.2.6.1.94i.SPA'
         image_file_location: 'tftp://10.89.0.11/installers/cisco-ftd-fp2k.6.4.0-1293.SPA'
         image_version: 6.4.0-1292
```
Comment (119Vik) [inline samples/ftd_install/install_ftd_on_2110_with_local_connection.yml]: let's add empty line here

File: test/unit/module_utils/test_device.py (hunk 1 above)
Comment (annikulin) [reply test/unit/module_utils/test_device.py]: Good point, moved.

File: samples/ftd_install/install_ftd_on_2110_with_local_connection.yml (hunk 2 above)
Comment (annikulin) [reply samples/ftd_install/install_ftd_on_2110_with_local_connection.yml]: Done.

**Pull request merged**